NumberValidator/Helpers/LuhnAlgorithm.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace NumberValidator.Helpers
+{
+    public class Luhn
```

Comment:
if this `Luhn` algo is not used anywhere else then it should be inside the `IN` validators folder not in helper. Search in python repo if Luhn is reused. if reused then we can keep it in helper.

Reply:
Luhn algo is used in many countries so that i kept in helper folder